avail/src/main/kotlin/avail/anvil/AvailWorkbench.kt

```diff
@@ -1973,6 +1973,16 @@ class AvailWorkbench internal constructor(
 				projectManager?.onWorkbenchClose(this@AvailWorkbench)
 					?: exitProcess(0)
 			}
+
+			override fun windowClosing(e: WindowEvent?)
```

Comment:
We should probably strengthen the (unused) parameter type to `WindowEvent`, since Swing will never actually pass a `null` here. It's pedantic, in this case, but it's good to control the flow of `null`s as tightly as possible, and we would definitely want this to blow up if Swing did the wrong thing.